x/auth/vesting/types/vesting_account.go

```diff
@@ -820,8 +821,10 @@ func (va *ClawbackVestingAccount) computeClawback(clawbackTime int64) sdk.Coins
 	totalUnvested := sdk.NewCoins()
 	unvestedIdx := 0
```

Comment:
what does this mean?

Reply:
It's the index of the first period that's currently unvested. The clawback will be of this and subsequent periods.